Makefile

```diff
@@ -81,10 +82,26 @@ clean_sharness:
 	@rm -rf ./sharness/lib/sharness
 	@rm -rf sharness/trash\ directory*
 
+swagger: $(swagger)
+	@cd api/restapi && swagger generate spec -o ../../swagger/swagger.json 
+
+
+$(swagger):
+	@echo "Installing swagger"
+	@rm -rf swagger
+	@mkdir swagger
+	@go get -u github.com/go-swagger/go-swagger/cmd/swagger
+	@cd $$GOPATH/src/github.com/go-swagger/go-swagger && go install .
+
+clean_swagger:
+	@rm -rf swagger
+	@rm -rf $$GOPATH/src/github.com/go-swagger
```

Comment:
This messes too much with the system. It removes swagger, which could have been installed outside cluster. I think probably Makefile should not remove things outside its own project that he cannot be sure are owned by it.